docs/platform-support.rst

```diff
@@ -19,4 +19,91 @@
 
 ****************
 Platform Support
-****************
\ No newline at end of file
+****************
+
+*******************
+System Requirements
+*******************
+
+* NVIDIA RDMA-capable network adapters:
+   * NVIDIA ConnectX NICs
+        * ConnectX-5 or newer
+   * NVIDIA BlueField Network Platforms
+        * BlueField-2 DPU (NIC mode)
+        * BlueField-3 DPU (NIC mode)
+        * BlueField-3 SuperNIC (NIC mode)
+* NVIDIA GPU Operator Version 23.9.x or newer (required for the workloads using NVIDIA GPUs and GPUDirect RDMA technology)
+* Operating Systems:
+    * Ubuntu: v22.04, v20.04
+    * OpenShift Container Platform (OCP): v 4.14, v4.13, v4.12
+    * RHEL: v9.2, v9.1, v8.8, v8.6
+* Container runtime: containerd, CRI-O
+
+***********************
+Tested Network Adapters
+***********************
+The following network adapters have been tested with the Network Operator:
+
+* ConnectX-6 Dx
+* ConnectX-7
+* BlueField-2 NIC Mode
+* BlueField-3 NIC Mode
+
+*************
+Prerequisites
+*************
+
+.. list-table::
+   :header-rows: 1
+   * - Component
+     - Version
+     - Notes
+   * - Kubernetes
+     - >=1.24 and <=1.27
+     -  -
+   * - Helm
+     - v3.5+
+     - For information and methods of Helm installation, please refer to the official Helm Website.
+
+******************
+Component Versions
+******************
+The following component versions are deployed by the Network Operator:
+
+
+.. list-table::
+   :header-rows: 1
+
+   * - Component
+     - Version
+     - Notes
+   * - Node Feature Discovery
+     - v0.13.2
+     - Optionally deployed. May already be present in the cluster with proper configuration.
+   * - NVIDIA MLNX_OFED driver container
+     - 23.10-0.5.5.0
```

Comment:
Please use vars for component versions. e.g `|mofed-version|`

Need also to add `.. include:: ./common/vars.rst` after `... headings`